lib/portage/dbapi/vartree.py

```diff
@@ -4166,15 +4152,17 @@ def _security_check(self, installed_instances):
         if not suspicious_hardlinks:
             return 0
 
-        msg = []
-        msg.append(_("suid/sgid file(s) " "with suspicious hardlink(s):"))
-        msg.append("")
+        msg = [_("suid/sgid file(s) with suspicious hardlink(s):"), ""]
         for path_list in suspicious_hardlinks:
             for path, s in path_list:
-                msg.append("\t%s" % path)
-        msg.append("")
-        msg.append(
-            _("See the Gentoo Security Handbook " "guide for advice on how to proceed.")
+                msg.append(f"\t{path}")
+        msg.extend(
+            (
+                "",
+                _(
+                    "See the Gentoo Security Handbook guide for advice on how to proceed."
+                ),
+            )
```

Comment:
This might technically be more optimal, but I don't think that justifies the readability hit?